extensions/amp-samba-player/0.1/amp-samba-player.js

```diff
@@ -0,0 +1,260 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {isLayoutSizeDefined} from '../../../src/layout';
+import {getDataParamsFromAttributes,removeElement} from '../../../src/dom';
+import {
+  installVideoManagerForDoc,
+} from '../../../src/service/video-manager-impl';
+import {videoManagerForDoc} from '../../../src/video-manager';
+import {VideoEvents} from '../../../src/video-interface';
+import {tryParseJson} from '../../../src/json';
+import {user} from '../../../src/log';
+
+/** @const {string} */
+const TAG = 'amp-samba-player';
+
+/** @private {Object} */
+const API_MAP = {
+  'web4-7091': '//playerapitest2.liquidplatform.com:7091/embed/',
+  'localhost-8080': '//localhost:8080/player-api/embed/',
```

Comment:
do you need to support localhost and staging for you customers? If this was just for your testing, I recommend only keeping prod configs for the public facing API.